Specs/DateTime/French/HolidayExtractor.json

```diff
@@ -141,5 +141,130 @@
         "Length": 14
       }
     ]
+  },
+  {
+    "Input": "J'y retournerai à Noël",
+    "Results": [
```

Comment:
Same as previous PRs. Please add the NotSupported tags.

Reply:
If this build doesn't fail, you can skip adding in this one. But please add them in all future PRs.